apps/ledger-live-desktop/.eslintrc.js

```diff
@@ -59,9 +59,6 @@ module.exports = {
         "plugin:@typescript-eslint/recommended",
       ],
       parser: "@typescript-eslint/parser",
-      parserOptions: {
-        project: "./src/tsconfig.json",
-      },
```

Comment:
Hi @sprohaszka-ledger, can you please explain this change?
This will affect all of Live Desktop right?

Reply:
Yep same I would like to understand this change

Reply:
This was a change I introduced previously. It affects only TS code in desktop (which means only this PR).
I try to fix an issue before @elbywan find the right thing to do in  `.flowconfig` file.
Keeping it rise an warning at the first import statement.